multiaccounts/settings/database.go

```diff
@@ -149,10 +149,11 @@ INSERT INTO settings (
   wallet_display_assets_below_balance,
   wallet_display_assets_below_balance_threshold,
   wallet_collectible_preferences_group_by_collection,
-  wallet_collectible_preferences_group_by_community
+  wallet_collectible_preferences_group_by_community,
+	last_used_wallet_account_name_index_suggestion
 ) VALUES (
 ?,?,?,?,?,?,?,?,?,?,?,?,?,?,
-?,?,?,?,?,?,?,?,?,'id',?,?,?,?,?,?,?,?,?,?,?,?)`,
+?,?,?,?,?,?,?,?,?,'id',?,?,?,?,?,?,?,?,?,?,?,?,?)`,
```

Comment:
what is this code? 😅

Reply:
That's how we set initial settings, it's there for ages. 😃 
I suggested at some point in the past that we could refactor settings table to make it growing in height, not in width, but...

Reply:
These are all positional args? Can they not be named args so we would at least know what is being set?

Reply:
No, that's part of query. Look at the code.

Reply:
Surely could have a helper function to build that query which does this though?

Reply:
if only we had invented the ORM for such simple query :trollface: